parser/src/main/java/com/linkedin/avroutil1/util/ConfigurableAvroSchemaComparator.java

```diff
@@ -337,7 +340,22 @@ private static void recordSchemaEquals(AvroRecordSchema a, AvroRecordSchema b, S
         equals(aField.getSchema(), bField.getSchema(), config, seen, differences, fastFail);
 
         if (aField.hasDefaultValue() && bField.hasDefaultValue()) {
-          if (!aField.getDefaultValue().equals(bField.getDefaultValue())) {
+          if (AvroType.BYTES.equals(aField.getSchema().type())) {
```

Comment:
Does this issue come up for other comparators or just this one?

Reply:
Added a UT for apache Avro comparator. It is not an issue there.